From 510d031ee6b584342cee76b4dc3565bbafccf62d Mon Sep 17 00:00:00 2001
From: Vitaly Margolin <vmargolin@habana.ai>
Date: Mon, 16 Jun 2025 14:18:01 +0300
Subject: [PATCH] [SW-232431] lfs: add new fit load type to clean logs

Add new FIT load type to clean littleFS based
flash log

Change-Id: I518f69339041a77005da3ff22eb73b00f7b56c49
Signed-off-by: Vitaly Margolin <vmargolin@habana.ai>
---

diff --git a/fit_load_type.h b/fit_load_type.h
index eb28596..6690cd2 100644
--- a/fit_load_type.h
+++ b/fit_load_type.h
@@ -51,7 +51,8 @@
 	FIT_LOAD_TYPE_READ_LOG = 38,
 	FIT_LOAD_TYPE_WRITE_RACK_EEPROM = 39,
 	FIT_LOAD_TYPE_BIS_E_COMPLIANT_ENABLE = 40,
-	FIT_LOAD_TYPE_COUNT = 41,
+	FIT_LOAD_TYPE_CLEANUP_LOG = 41,
+	FIT_LOAD_TYPE_COUNT,
 };
 
 #endif /* FIT_LOAD_TYPE_H */
diff --git a/gpu/common/fit_load_type.h b/gpu/common/fit_load_type.h
index e8a2f56..117b8ab 100644
--- a/gpu/common/fit_load_type.h
+++ b/gpu/common/fit_load_type.h
@@ -51,6 +51,7 @@
 	FIT_LOAD_TYPE_READ_LOG = 38,
 	FIT_LOAD_TYPE_WRITE_RACK_EEPROM = 39,
 	FIT_LOAD_TYPE_BIS_E_COMPLIANT_ENABLE = 40,
+	FIT_LOAD_TYPE_CLEANUP_LOG = 41,
 	FIT_LOAD_TYPE_COUNT,
 };
 
